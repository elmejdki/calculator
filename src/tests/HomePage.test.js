import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from '../components/pages/HomePage';

it('renders correctly the Home Page', () => {
  const tree = renderer.create(<HomePage />).toJSON();

  expect(tree).toMatchSnapshot();
});
