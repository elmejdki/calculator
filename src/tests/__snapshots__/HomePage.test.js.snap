// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the Home Page 1`] = `
<div
  className="home max-height max-width"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Excepteur do nisi irure nisi eiusmod minim reprehenderit. Qui do do mollit sint ad sunt. Sunt in tempor esse veniam. Tempor nostrud commodo deserunt nisi Lorem sunt. In anim voluptate nostrud eu amet.
  </p>
  <p>
    Esse in aliquip ullamco ad amet duis magna mollit ex. Quis dolore adipisicing non quis. Sunt id adipisicing laborum velit aute laborum sit sunt dolor duis labore occaecat consectetur consequat. Anim pariatur sunt enim ea incididunt. Aliquip cillum consectetur veniam ullamco consequat in qui.
  </p>
  <p>
    Minim non dolor et consectetur reprehenderit id ad dolor occaecat. Id qui magna labore anim. Duis ad ullamco esse sit deserunt Lorem officia. Cillum magna elit et aute consectetur ut deserunt ex eiusmod consectetur laborum incididunt excepteur. Tempor mollit fugiat quis laborum culpa qui irure do reprehenderit ullamco. Magna adipisicing cupidatat magna est do nostrud mollit ullamco sunt ullamco. Eu sint Lorem mollit pariatur nostrud aute occaecat.
  </p>
  <p>
    Dolor proident consectetur tempor culpa deserunt Lorem mollit proident. Cillum amet dolore exercitation occaecat incididunt. Cillum deserunt ad dolor sunt irure culpa non culpa ea id ea. Adipisicing velit voluptate ex eu. In aliqua ut fugiat aliqua nostrud culpa veniam eu laborum amet ex occaecat ullamco dolore.
  </p>
</div>
`;
